tel2puml/pv_event_simulator.py

```diff
@@ -299,7 +299,7 @@ def transform_dict_into_pv_event(
         applicationName=pv_event_model.applicationName,
         jobName=pv_event_model.jobName,
     )
-    if len(pv_event_model.previousEventIds) > 0:
+    if mapping_config.previousEventIds in pv_dict:
```

Comment:
rather than doing this, because we now have a default of an empty list on PVEventModel just set previousEventIds to pv_event_model.previousEventIds